dynawo/sources/Models/Modelica/PreassembledModels/NrtModels.txt

```diff
@@ -6,6 +6,7 @@ CurrentLimitAutomaton
 CurrentLimitAutomatonTwoLevels
 DistanceProtectionLineTwoZones
 DoubleStep
+ElectronicLoad
```

Comment:
not related

Reply:
no need to make a dedicated commit to fix such thing